hendrix/utils/__init__.py

```diff
@@ -7,6 +7,13 @@
 
 HENDRIX_DIR = os.path.dirname(os.path.abspath(__file__))
 
+# Default PID Directory
+PID_DIR = os.path.join('/', 'var', 'tmp', 'hendrix')
```

Comment:
Maybe, `os.path.join(tempfile.tempdir, 'hendrix')`?
A better way would be to use `tempfile.mkdtemp()`, however it should be executed before forking to not end up with a different directory for every instance

Reply:
:heavy_check_mark: